src/ompl/geometric/planners/bitstar/BITstar.h

```diff
@@ -0,0 +1,614 @@
+            void setKNearest(bool useKNearest);
+
+            /** \brief Get whether a k-nearest search is being used.*/
+            bool getKNearest() const;
+
+            /** \brief Enable tracking of failed edges. This currently is too expensive to be useful.*/
+            void setUseFailureTracking(bool trackFailures);
+
+            /** \brief Get whether a failed edge list is in use.*/
+            bool getUseFailureTracking() const;
+
+            /** \brief Enable "strict sorting" of the edge queue.
+            Rewirings can change the position in the queue of an edge.
+            When strict sorting is enabled, the effected edges are resorted
+            immediately, while disabling strict sorting delays this
+            resorting until the end of the batch. */
+            void setStrictQueueOrdering(bool beStrict);
+
+            /** \brief Get whether strict queue ordering is in use*/
+            bool getStrictQueueOrdering() const;
+
+            /** \brief Enable pruning of vertices/samples that CANNOT improve the current solution.
+            When a vertex in the graph is pruned, it's descendents are also pruned
+            (if they also cannot improve the solution) or placed back in
+            the set of free samples (if they could improve the solution).
+            This assures that a uniform density is maintained.*/
+            void setPruning(bool prune);
+
+            /** \brief Get whether graph and sample pruning is in use.*/
+            bool getPruning() const;
+
+            /** \brief Set the fractional change in the solution cost necessary for pruning to occur. */
+            void setPruneThresholdFraction(double fractionalChange);
+
+            /** \brief Get the fractional change in the solution cost necessary for pruning to occur. */
+            double getPruneThresholdFraction() const;
+
+            /** \brief Delay considering rewiring edges until an initial solution is found. This improves
+            the time required to find an initial solution when doing so requires multiple batches and has
+            no effects on theoretical asymptotic optimality (as the rewiring edges are eventually considered). */
+            void setDelayRewiringUntilInitialSolution(bool delayRewiring);
+
+            /** \brief Get whether BIT* is delaying rewiring until a solution is found. */
+            bool getDelayRewiringUntilInitialSolution() const;
+
+            /** \brief Stop the planner each time a solution improvement is found. Useful
+            for examining the intermediate solutions found by BIT*. */
+            void setStopOnSolnImprovement(bool stopOnChange);
+
+            /** \brief Get whether BIT* stops each time a solution is found. */
+            bool getStopOnSolnImprovement() const;
+            ///////////////////////////////////////
+
+            ///////////////////////////////////////
+            // Planner progress property functions
```

Comment:
Yes, it does work. We should really change that in RRT*, I realized that was what you were following. Anyway, no big deal :)